src/main/webapp/app/exercises/programming/shared/commits-info/commits-info-group/commits-info-row/commits-info-row.component.html

```diff
@@ -0,0 +1,74 @@
+<tr class="{{ isExpanded ? (groupCommitIndex === groupCommitCount - 1 ? 'bg-light border-bottom' : 'bg-light') : '' }}">
+    <th scope="row" class="p-2">
+        @if (groupCommitIndex === 0) {
+            {{ pushNumber }}
+        } @else {
+            <span class="text-body-secondary">
+                {{ pushNumber + '.' + (groupCommitIndex + 1) }}
+            </span>
+        }
+    </th>
+    <td class="p-2">
+        @if (localVC) {
+            @if (isRepositoryView) {
+                <a [routerLink]="['./', commit.hash]" ngbTooltip="{{ commit.hash }}"> {{ commit.hash ?? '' | truncate: 7 }} </a>
+            } @else {
+                <span ngbTooltip="{{ commit.hash }}">
+                    {{ commit.hash ?? '' | truncate: 7 }}
+                </span>
+            }
+        } @else {
+            <a [href]="commit.commitUrl" ngbTooltip="{{ commit.hash }}"> {{ commit.hash ?? '' | truncate: 7 }} </a>
+        }
+    </td>
+    <td class="p-2">
+        @if (isRepositoryView) {
+            {{ user?.name ?? commit.author }}, {{ user?.email ?? commit.authorEmail }}
```

Comment:
Why is it better to use the user that is assigned to the participation? Even if the commit author can be manipulated, I'd rather see as a student or instructor that there is something wrong than simply see the user that Artemis assumes was the author

Reply:
I agree. Especially since (If I understand that correctly) the initial commit containing the template will then also have student as a autor, which is wrong.

Reply:
Yes, that makes more sense. In the original issue this was the preferred solution, but we discussed this with @maximiliansoelch and decided to keep it based on the git user.